.travis.yml

```diff
@@ -17,6 +17,9 @@ cache:
         - $HOME/.stack
 
 matrix:
+  allow_failures:
+    - env: BUILD=qa
```

Comment:
http://stackoverflow.com/a/35758162/53974 confirms you need `BUILD=qa GHCVER=8.0.1 STACK_YAML=stack-8.0.yaml?` (as value for `env:`).